css/index.css

```diff
@@ -1,3 +1,635 @@
+	flex-direction: column;
+	justify-content: flex-start;
+
+}
+
+span{
+	display: flex;
+	width: 100%;
+	justify-content: center;
+	align-items: center;
+	margin: 10px;
+	padding: 10px;
+	align-self: center;
+}
+
+
+footer p{
+	line-height: 25px;
+}
+
+footer address{
+	padding-bottom: 20px;
+}
+
+footer h2{
+	font-size: 24px;
+}
+
+.Services-nav nav{
+	display: flex;
+	padding: 10px;
+	justify-content: space-around;
+	align-items:center;
+	flex-wrap: wrap;
+}
+.Services-nav a{
+	padding:5px;
+	margin:5px;
+	text-decoration: none;
+	color: gray;
+	box-sizing: border-box;
+	text-align: : center;
+	font-size:16px;
+}
+
+.Services-nav img{
+	padding: 5px;
+	margin:5px;
+}
+
+.Hero-section-services .services-header-img{
+	height: 300px;
+	width: 1300px;
+	display: flex;
+	justify-content: space-around;
+	align-items: center;
+}
+
+.Hero-section-services .hero-copy-services{
+	display: flex;
+	flex-direction: column;
+	justify-content:center;
+	width: 1300px;
+	margin-top: 20px;
+}
+.Hero-section-services .hero-copy-services h1{
+	font-size: 36px;
+}
+
+.Hero-section-services{
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+	height: 450px;
+	border-bottom: 2px solid black;
+	margin: 25px;
+	padding: 25px;
+}
+
+.Design {
+	display: flex;
+	justify-content: space-around;
+	align-items: center;
+	height: 300px;
+	margin: 10px;
+	padding: 10px;
+	margin-bottom: 40px;
+	margin-top: 40px;
+}
+.Design .digital_design {
+	border: 1px solid black;
+	background-color: lightgray;
+	width: 600px;
+	height: 300px;
+}
+
+.Design .UX_UI{
+	border: 1px solid black;
+	background-color: lightgray;
+	width: 600px;
+	height: 300px;
+}
+
+.Design h2{
+	font-size: 24px;
+	margin: 10px;
+	padding:10px;
+}
+
+.Design p{
+	font-size: 16px;
+	margin: 10px;
+	padding: 10px;
+}
+
+.Design button{
+	width: 150px;
+	font-size:16px;
+	height: 50px;
+	background: white;
+	border-style: solid;
+	border-radius: 2px;
+	border-color: black;
+	margin: 10px;
+	padding: 10px;
+
+}
+
+
+
+.Marketing {
+	display: flex;
+	justify-content: space-around;
+	align-items: center;
+	height: 300px;
+	margin: 10px;
+	padding: 10px;
+	margin-bottom: 40px;
+	margin-top: 20px;
+}
+.Marketing .digital_marketing {
+	border: 1px solid black;
+	background-color: lightgray;
+	width: 600px;
+	height: 300px;
+}
+
+.Marketing .Web-dev{
+	border: 1px solid black;
+	background-color: lightgray;
+	width: 600px;
+	height: 300px;
+}
+
+.Marketing h2{
+	font-size: 24px;
+	margin: 10px;
+	padding:10px;
+}
+
+.Marketing p{
+	font-size: 16px;
+	margin: 10px;
+	padding: 10px;
+}
+
+.Marketing button{
+	width: 150px;
+	font-size:16px;
+	height: 50px;
+	background: white;
+	border-style: solid;
+	border-radius: : 2px;
+	border-color: black;
+	margin: 10px;
+	padding: 10px;
+
+}
+
+
+.App-dev {
+	display: flex;
+	justify-content: space-around;
+	align-items: center;
+	height: 300px;
+	margin: 10px;
+	padding: 10px;
+	margin-bottom: 40px;
+	margin-top: 20px;
+}
+.App-dev .ios {
+	border: 1px solid black;
```

Comment:
Next time, use comments to specify if a part of the CSS is being applied to the another page (_since this CSS file is providing styles to two pages, `index` and `about`_). That would make this more readable.